src/sonic-yang-models/tests/yang_model_tests/tests_config/mgmt_port.json

```diff
@@ -0,0 +1,65 @@
+{
+    "MGMT_PORT_TEST": {
+        "sonic-mgmt-port:sonic-mgmt-port": {
+            "sonic-mgmt-port:MGMT_PORT": {
+                "MGMT_PORT_LIST": [
+                    {
+                        "alias": "eth0",
+                        "name": "eth0",
+                        "admin_status": "up",
+                        "speed": 100,
+                        "autoneg": true,
+                        "description": "management interface",
```

Comment:
This field is 'on'/'off' as per backend.

                /* Set autoneg and ignore the port speed setting */
                else if (fvField(i) == "autoneg")
                {
                    an = (fvValue(i) == "on");
                }

Reply:
@praveen-li . We use boolean for autoneg because the PORT_TABLE and MGMT_PORT_TABLE config DB schema use BIT for autoneg (https://github.com/Azure/sonic-swss/blob/master/doc/swss-schema.md). 
I couldn't find your code, is it still in a PR? 

Thanks,

Reply:
Never mind. Got the recent commit for it from Joyas.

Reply:
Done. Changed to "on|off"